packages/yoastseo/spec/scoring/assessments/inclusiveLanguage/configuration/ageAssessmentsSpec.js

```diff
@@ -82,4 +82,70 @@ describe( "Age assessments", function() {
 		expect( isApplicable ).toBeFalsy();
 		expect( assessor.getMarks() ).toEqual( [] );
 	} );
+
+	it( "correctly identifies a phrase that is only recognized when followed by participle or simple past tense", () => {
+		const mockPaper = new Paper( "The aged worked, the better they are." );
+		const mockResearcher = Factory.buildMockResearcher( [ "The aged worked, the better they are." ] );
+		const assessor = new InclusiveLanguageAssessment( assessments.find( obj => obj.identifier === "theAged" ) );
```

Comment:
This is not really an assessor, but rather an individual assessment. However, I noticed that the other tests also use the `assessor` term 🤔 .